src/couch/src/couch_proc_manager.erl

```diff
@@ -290,6 +286,9 @@ handle_config_change("native_query_servers", _, _, _, _) ->
 handle_config_change("query_server_config", _, _, _, _) ->
     gen_server:cast(?MODULE, reload_config),
     {ok, undefined};
+handle_config_change("couchdb", "js_engine", _, _, _) ->
```

Comment:
Maybe out of scope of this PR, but we might want to handle `handle_config_change("native_query_servers", _, _, _, _)` in a similar way.

Reply:
> Maybe out of scope of this PR, but we might want to handle `handle_config_change("native_query_servers", _, _, _, _)` in a similar way.

We already do, it's just above in lines 283:285